proto/minder/v1/minder.proto

```diff
@@ -342,6 +342,17 @@ service RepositoryService {
             relation: RELATION_REPO_CREATE
         };
     }
+    rpc ReconcileEntityRegistration(ReconcileEntityRegistrationRequest) returns (ReconcileEntityRegistrationResponse) {
```

Comment:
One more thing, because this RPC can register any types, shouldn't it be part of the provider service? (See also its http path..)

Reply:
I agree, it's better placed under provider service.

Reply:
I thought about this at the beginning as well but then I noticed that the provider service contains RPCs explicitly related to CRUD operations for the _provider_ entity itself. Indeed, the repository service is not a suitable place.
Still, I've moved this to the Provider service as it's the closest logically related one.